bigtest/network/factories/package.js

```diff
@@ -74,6 +74,27 @@ export default Factory.extend({
     }
   }),
 
+  withProxy: trait({
+    afterCreate(resource, server) {
+      let proxy = server.create('proxy', {
+        inherited: true,
+        id: 'bigTestJS'
+      });
+      resource.update('proxy', proxy.toJSON());
+      resource.save();
+    }
+  }),
+
+  withInheritedProxy: trait({
+    afterCreate(resource, server) {
+      let proxy = server.create('proxy', {
+        inherited: true,
+        id: 'bigTestJS'
+      });
+      resource.update('proxy', proxy.toJSON());
+      resource.save();
+    }
```

Comment:
Do we really need both of these traits, specially since they are the same or can we get rid of one?

Reply:
Adjusting the withProxy to reflect inherited = false and id = microstates so they are distinct

Reply:
is there a specific reason why you chose to use the work `resource` carole? The reason I am asking is that in David's PR, he is using pkg - since its for package - if we change this, we could reduce number of conflicts - please let me know what you think!

Reply:
Updated to use `packageObj` so it is consistent with others -- good catch